src/assets/translations/fr/main.json

```
@@ -75,6 +75,8 @@
     "retryQuestion": "Réessayer ?",
     "transactionFailed": "Échec de la transaction",
     "transactionFailedBody": "Échec de la transaction",
+    "safeProposalQueued": "Proposition SAFE en file d'attente. %{currentConfirmations} sur %{confirmationsRequired} signées.",
```

Would deviate a bit from the en variant here, as `x out of y signed` here means `number of signers that have signed` in this context
```suggestion
    "safeProposalQueued": "Proposition SAFE en attente. Signatures: %{currentConfirmations} sur %{confirmationsRequired}.",
```

---

Great, minor nitpick there should always be a space before ":" in French (not and never in English).

```suggestion
    "safeProposalQueued": "Proposition SAFE en attente. Signatures : %{currentConfirmations} sur %{confirmationsRequired}.",
```